content/03.results.md

```diff
@@ -204,37 +199,36 @@ The output from `SingleR` includes a score matrix containing a score for each ce
 <!-- TODO: ⚠️ For review - What do you think of the next sentence? -->
-Using this measure, we found that the `BlueprintEncodeData` reference [@doi:10.3324/haematol.2013.094243; @doi:10.1038/nature11247], which includes a variety of normal cell types, tended to perform better than or at least similarly to other references across samples from different disease types (Figure {@fig:figS4}).
+Using this measure, we found that the `BlueprintEncodeData` reference [@doi:10.3324/haematol.2013.094243; @doi:10.1038/nature11247], which includes a variety of normal cell types, tended to perform similarly to or better than other references when applied to samples from different disease types (Figure {@fig:figS4}).
 Based on these findings, we used the `BlueprintEncodeData` reference to annotate cells from all libraries on the Portal.
 Use of a consistent reference also supports cross-project analyses.
 
 In contrast, `CellAssign` is a marker-gene-based annotation method that requires a binary matrix with all cell types and all associated marker genes as the reference.
 We used the list of marker genes available as part of `PanglaoDB` [@doi:10.1093/database/baz046] to construct organ-specific marker gene matrices with marker genes from all cell types listed for the specified organ.
 Since many cancers may have infiltrating immune cells, all immune cells were also included in each organ-specific reference.
-For each ScPCA project, we provided the organ-specific marker gene matrix relevant to the disease and tissue type from which the sample was obtained (e.g., for brain tumors, we used a brain-specific marker gene matrix with all brain and immune cell types).
+For each ScPCA project, we used the organ-specific marker gene matrix relevant to the disease and tissue type from which the sample was obtained (e.g., for brain tumors, we used a brain-specific marker gene matrix with all brain and immune cell types).
 If `CellAssign` cannot find a likely cell type from the marker gene matrix, it does not assign a cell type.
-Because we annotate cells from tumor samples using references containing only normal cells, we anticipate that many cells, particularly the tumor cells, may not have an exact match; reporting this to the end user is valuable.
+Because we annotate cells from tumor samples using references containing only normal cells, we anticipate that many cells, particularly the tumor cells, will not have an exact match; reporting this to the end user is valuable.
 Indeed, when applying `CellAssign` to tumor samples with our chosen reference, we observed that many cells were unassigned.
 We included an example in Figure {@fig:figS5}A where unassigned cell types are labeled with `Unknown`.
```

Comment:
The phrase "reporting this to the end user is valuable" feels a bit terse. I was working to rephrase this but wondered if we should actually just delete it? I'm not sure this info contributes much more than saying we report these with an `Unknown` label.